index.html

```diff
@@ -110,7 +110,7 @@ <h1 class="font-weight-bold mb-4 text-white hero-8-title">
                 Tech Talk 7.0
               </h1>
               <p class="mb-5 text-white-70 w-lg-75 mx-auto">
-                Secrets on how to get into the BIG 5 companies as a developer.
+                <span>Secrets on how to get into the BIG 5 companies as a developer.</span> 

```

Comment:
Please why did you add a span to this?

Reply:
Wanted to make the sentence a bit bolder in the css file,  it was a bit difficult to see considering it's the first sentence you read once you visit the site

Reply:
That's great. Please effect the changes and create another PR

Reply:
Kk

Reply:
OK.